src/main/java/uk/gov/hmcts/divorce/common/notification/Applicant2AppliedForFinalOrderNotification.java

```diff
@@ -51,24 +54,44 @@ public void sendToApplicant2(final CaseData caseData, final Long id) {
 
     @Override
     public void sendToApplicant1Solicitor(CaseData caseData, Long caseId) {
-        if (!caseData.getApplicationType().isSole()
-            && YES.equals(caseData.getFinalOrder().getApplicant2AppliedForFinalOrderFirst())) {
-            log.info("Sending Applicant 1 notification informing them that other party have applied for final order: {}", caseId);
-            notificationService.sendEmail(
-                caseData.getApplicant1().getSolicitor().getEmail(),
-                JOINT_SOLICITOR_OTHER_PARTY_APPLIED_FOR_FINAL_ORDER,
-                commonContent.solicitorTemplateVars(caseData, caseId, caseData.getApplicant1()),
-                ENGLISH
-            );
+        if (!caseData.getApplicationType().isSole()) {
+            if (Objects.isNull(caseData.getFinalOrder().getApplicant1AppliedForFinalOrder())) {
+                log.info("Sending Applicant 1 notification informing them that other party have applied for final order: {}", caseId);
+                notificationService.sendEmail(
+                        caseData.getApplicant1().getSolicitor().getEmail(),
+                        JOINT_SOLICITOR_OTHER_PARTY_APPLIED_FOR_FINAL_ORDER,
+                        commonContent.solicitorTemplateVars(caseData, caseId, caseData.getApplicant1()),
+                        caseData.getApplicant1().getLanguagePreference()
+                );
+            } else {
+                sendBothSolicitorsAppliedForFinalOrderNotification(caseData, caseId, caseData.getApplicant1());
+            }
+        }
+    }
+
+    @Override
+    public void sendToApplicant2Solicitor(CaseData caseData, Long caseId) {
+        if (!caseData.getApplicationType().isSole() && Objects.nonNull(caseData.getFinalOrder().getApplicant2AppliedForFinalOrder())) {
```

Comment:
and this check should be for applicant 1

>Objects.nonNull(caseData.getFinalOrder().getApplicant2AppliedForFinalOrder())

Reply:
Reverted the change as spotted by Sunil